components/molecules/PaymentForm/PaymentForm.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext, useState, useEffect } from 'react';
+import StripePayment from './../Stripe';
+// import PaymentStub from './../PaymentStub'
```

Comment:
do you need this commented code here?

Reply:
This is required in case you want to switch to a generic payment interface. Removing the comments will make it harder. But you can pull this comment into a list of instructions for the case you need to remove stripe lib ( platform independence )